packages/truffle-db/src/workspace/pouch.ts

```diff
@@ -90,68 +87,24 @@ export class Workspace {
     await this.ready;
 
     const { contracts } = input;
-    
+
     return {
       contracts: Promise.all(contracts.map(
         async (contractInput) => {
-          const { name, source } = contractInput;
-          const id = name !== undefined ? soliditySha3(name, source.id) : soliditySha3(source.id);
+          const { name, abi, compilation, sourceContract, constructor: contractConstructor } = contractInput;
```

Comment:
Stylistic note: until/unless we turn on prettier for TypeScript, I worry that it might get more difficult to read this code with long lines. General style rule of thumb here is to put each variable on its own line:
```suggestion
          const {
            name,
            abi,
            compilation,
            sourceContract,
            constructor: contractConstructor
          } = contractInput;
```